assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/stable_diffusion/convert.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import mlflow
+from mlflow.models import ModelSignature
+from mlflow.types.schema import ColSpec
+from mlflow.types.schema import DataType, Schema
+import os
+from pathlib import Path
+import shutil
+import sys
+from typing import Dict
+
+from .constants import ColumnNames, MLflowLiterals, MLflowSchemaLiterals, Tasks
+from azureml.model.mgmt.config import ComponentConstants
+from azureml.model.mgmt.utils.common_utils import log_execution_time
+
+
+
+def _get_mlflow_signature(task_type: str) -> ModelSignature:
+    """Return MLflow model signature with input and output schema given the input task type.
+
+    :param task_type: Task type used in training
+    :type task_type: str
+    :return: MLflow model signature.
+    :rtype: mlflow.models.signature.ModelSignature
+    """
+
+    if task_type in [Tasks.TEXT_TO_IMAGE.value]:
+        input_schema = Schema(
+            [ColSpec(MLflowSchemaLiterals.STRING_DATA_TYPE, ColumnNames.TEXT_PROMPT)]
+        )
+    else:
+        raise NotImplementedError(f"Task type: {task_type} is not supported yet.")
+
+    output_schema = Schema(
+        [ColSpec(MLflowSchemaLiterals.STRING_DATA_TYPE, ColumnNames.TEXT_PROMPT),
```

Comment:
don't return prompt in output. It will have duplicate column in batch deployment.

Reply:
Thanks for bringing this up. As we discussed we will handle batch output separately.